python/README.md

```diff
@@ -1,50 +1,85 @@
-# Sample Python apps  
-Here you will find a number of a sample Python applications that demonstrate using various features of the Couchbase Cloud Public API to automate operations. 
-
-To use any of these samples, follow the instructions below
+# Examples of using Public API with Python  
+Here we have a selection of Python3 scripts that shows each of the Public APIs being used.  These are not exhaustive and are definitely _not_ for production use.  They are merely here to get you started and, hopefully, to inspire you to come up with interesting ways to use the Public API that we've created.   
 
 
 ## Prerequisites
-The following pieces need to be in place
+The following pieces need to be in place before you try the samples :-
 
-* Created Access and Secret Keys for the API
-* Python 3.8 or greater
+* Created by using the Capella UX,  API Access and Secret Keys
+* Python 3.9 or greater
 
 
 ## Running 
 You will need to download the samples. To do this, either download [the archive](https://github.com/couchbasecloud/rest-api-examples.git) or clone the repository:
 
 ```
-$ git clone https://github.com/couchbasecloud/rest-api-examples.git
+git clone https://github.com/couchbasecloud/rest-api-examples.git
 ```
 
 change directory
 ```
-$ cd rest-api-examples/python
+cd rest-api-examples/python
 ```
 
 
 There are several Python libraries that will need to be installed and these are listed in _requirements.txt_.  They can be automatically loaded using the _pip_ command:
 ```
-$ pip install -r requirements.txt
+pip install -r requirements.txt
+```
+
+Depending on how you have Python installed, you may need to call pip for Python3 like this
```

Comment:
python3 -m pip install -r requirements.txt
...should work on any platform, smoothing over differences in older OSes